src/controllers/check.your.answers.controller.ts

```diff
@@ -11,11 +11,19 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
   if (req.session) {
     try {
       const { companyName, companyNumber } = retrieveCompanyProfileFromObjectionSession(req.session);
+      let shareIdentity: string = "No";
       const objection: Objection = await getObjection(req.session);
+      if (objection) {
```

Comment:
this new code needs to be covered by unit test

Reply:
Updated tests to check that the fields are present.